src/Services/Processor/Substrate/Codec/Encoder.php

```diff
@@ -46,6 +46,11 @@ class Encoder
         'SetAttribute' => 'MultiTokens.set_attribute',
         'RemoveAttribute' => 'MultiTokens.remove_attribute',
         'RemoveAllAttributes' => 'MultiTokens.remove_all_attributes',
+        'AcceptCollectionTransfer' => 'MultiTokens.accept_collection_transfer',
+    ];
+
+    protected static array $overrideCallIndex = [
+        'MultiTokens.accept_collection_transfer' => [40, 41],
```

Comment:
Just curious, does this index not get loaded with all the others from the metadata?